nio/src/main/scala/zio/nio/file/Files.scala

```diff
@@ -74,6 +81,13 @@ object Files {
     effectBlocking(Path.fromJava(JFiles.createTempFile(prefix.orNull, suffix, fileAttributes.toSeq: _*)))
       .refineToOrDie[IOException]
 
+  def createTempFileManaged(
+    suffix: String = ".tmp",
+    prefix: Option[String],
```

Comment:
Should be fine to have `prefix` default to `None` and `fileAttributes` to default to `Iterable.empty`?